pykern/pkcli/ci.py

```diff
@@ -110,7 +110,7 @@ def run():
     check_eof_newline()
     check_main()
     check_prints()
-    fmt.diff(pkio.py_path())
+    fmt.diff(_paths(pkio.py_path()))
```

Comment:
`*_paths()` is the way to expand a list/tuple.

Reply:
81558f6e653269c07a7286f77898de84b03955fc